ui/push-health/Test.jsx

```diff
@@ -212,17 +212,19 @@ class Test extends PureComponent {
           >
             <Button
               onClick={this.toggleDetails}
-              className="text-break text-wrap border-0"
+              className="text-break text-wrap border-0 d-flex text-left"
```

Comment:
Some of the lines were centering themselves if they were too long.  So this fixes that and lets them wrap properly.